shared/templates/template_BASH_audit_rules_privileged_commands

```diff
@@ -3,7 +3,7 @@
 # Include source function library.
 . /usr/share/scap-security-guide/remediation_functions
 
-PATTERN="-a always,exit -F path=%PATH%.*"
+PATTERN="-a always,exit -F path=%PATH%\\s*.*"
```

Comment:
Is there a special reason why to use double backslash? I think that it is correct, but if you replace double quotes with single quotes, you can use single backslash.

Reply:
@matejak the regex wasn't properly handled without the double backslash. 

Should we replace the double quotes with single quotes? Not sure if we would come across other problems that we would then have to figure out.

Reply:
I have reviewed the called bash function and it is full of bugs. Fixing that one is out of scope of this PR, so I merge it as it works in this current form, and we can fix everything later, during a refactoring phase.
However, by looking at the function code, I am not able to explain why single backslash doesn't work and double does.